automerge-cli/Cargo.toml

```diff
@@ -10,8 +10,11 @@ path = "src/main.rs"
 
 [dependencies]
 clap = "3.0.0-beta.1"
+serde_json = "^1.0"
+anyhow = "1.0"
+atty = "0.2"
+duct = "0.13"
```

Comment:
Can we make this a dev dependency?

Reply:
oh, good call. didn't realize rust had the distinction!

Reply:
⚡ https://github.com/automerge/automerge-rs/pull/21/commits/08d79fefe00e5f3cf954f4bce7b26dda0b50f0ae